storage/storage_integration_tests/write_dir_changer.cpp

```diff
@@ -12,20 +12,9 @@ WritableDirChanger::WritableDirChanger(string testDir)
   , m_testDirFullPath(m_writableDirBeforeTest + testDir)
 {
   Platform & platform = GetPlatform();
-  Platform::EError const err = platform.MkDir(m_testDirFullPath);
-  switch (err)
-  {
-  case Platform::ERR_OK:
-    break;
-  case Platform::ERR_FILE_ALREADY_EXISTS:
-    Platform::EFileType type;
-    TEST_EQUAL(platform.GetFileType(m_testDirFullPath, type), Platform::ERR_OK, ());
-    TEST_EQUAL(type, Platform::FILE_TYPE_DIRECTORY, ());
-    break;
-  default:
-    TEST(false, ("Can't create directory:", m_testDirFullPath));
-    break;
-  }
+  platform.RmDirRecursively(m_testDirFullPath);
```

Comment:
What if, by mistake, during experiments, I'll put something important (like, big mwms) in a test dir, and this code will delete them?